index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My_Portfolio</title>
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.8.2/css/all.css"
+      integrity="sha384-oS3vJWv+0UjzBfQzYUhtDYW+Pj2yciDJxpsK1OYPAYjqT085Qq/1cq5FLXAZQ7Ay"
+      crossorigin="anonymous"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato&family=Poppins:ital@0;1&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css"
+      integrity="sha512-NhSC1YmyruXifcj/KFRWoC561YpHpc5Jtzgvbuzx5VozKpWvQ+4nXhPdFgmx8xqexRcpAglTj9sIBWINXa8x5w=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <main>
+      <header>
+        <div class="container-nav">
+          <a href="index.html" class="logo-title">Marlyn Mayienga</a>
+          <nav class="nav-bar">
+            <ul>
+              <li class="portfolio">
+                <a href="#portfolio">Portfolio </a>
+              </li>
+              <li>
+                <a href="#">About </a>
+              </li>
+              <li>
+                <a href="#contact">Contact </a>
+              </li>
+            </ul>
+          </nav>
+        </div>
+        <div class="hamburger-btn">
+            <i class="fas fa-bars"></i>
+          </div>
+      </header>
+      <section class="introduction">
+        <div class="container">
+        <div class="first-ellipse"></div>
+        <div class="ellipse_cut"></div>
+        <div class="second_img"></div>
+        <div class="first_img"></div>
+          <h1 class="greeting">I’m Merlin <br> Glad to see you!
+    </h1>
+          <p class="first-paragraph">
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
```

Comment:
While you did implement the previous reviewer's requirement, you didn't do so properly. The `font-weight` of the header is not quite matching that of the template design. You may need to import your Google Font differently in order to have various `font-weights` available for use. Refer to [this video](https://www.youtube.com/watch?v=mRoBBOTYNz4) to learn how to import fonts with different weights. Please fix the header.

Additionally, make sure the `<p>` and `<h1>` colors follow that of the template design. The `<h1>` color should be different from that of the `<p>`. Please fix this.